data/sql/updates/pending_db_characters/rev_1624468542752528700.sql

```diff
@@ -0,0 +1,4 @@
+INSERT INTO `version_db_characters` (`sql_rev`) VALUES ('1624468542752528700');
+
+ALTER TABLE `characters` DROP IF EXISTS `order`;
+ALTER TABLE `characters` ADD `order` TINYINT(1) NULL DEFAULT NULL AFTER `grantableLevels`;
```

Comment:
you can remove the (1) of TINYINT, it will be deprecated soon

Reply:
not "Can" -> "Should" :P